website/for_teachers.py

```diff
@@ -920,6 +920,7 @@ def create_accounts(self, user, class_id):
         Class = self.db.get_class(class_id)
         if not Class or (not utils.can_edit_class(user, Class) and not is_admin(user)):
             return utils.error_page(error=403, ui_message=gettext("no_such_class"))
+        # should this be split up in 403 and 401?
```

Comment:
I think this actually a case of a 404 and 401. Because there's a possibility that there isn't a class, and also that this person can't edit this class.